### PR TITLE
Added player health UI

### PR DESCRIPTION
**Pull request by redknight990**
- Can now see current HP on screen
- Fixed player & enemy HP to prevent from going below 0
- Renamed HealthDisplay to HealthUI

### CODE REVIEW

**Review by emilie-martin — CHANGES_REQUESTED** [review]
just a little indentation problem but otherwise it looks good!

**Review by daniela-venuta — APPROVED** [review]
Lovely!

File: Hush/Assets/Scripts/Player/Enums/PlayerAnimator.cs [hunk 1]
```diff
@@ -9,5 +9,16 @@ public static class PlayerAnimator
 		public static int HeavyAttack => Animator.StringToHash("HeavyAttack");
```
[the remark this answers is not in the document]
Comment (redknight990) [reply Hush/Assets/Scripts/Player/Enums/PlayerAnimator.cs]: fixed it!

**Review by emilie-martin — APPROVED** [review]
looks good!

**Pull request merged**